lib/ace/layer/cursor.js

```diff
@@ -69,6 +69,37 @@ var Cursor = function(parentEl) {
     };
     
 
+    this.$startCssAnimation = function() {
+        var blinkClassName = "ace_animate-blinking-" + this.blinkInterval;
+        if (!dom.hasCssString(blinkClassName)) {
+            var frames = 
+                "." + blinkClassName + "{\n" +
+                "    " + dom.CSS_KEYFRAMES_PREFIX + "animation: " + this.blinkInterval + "ms blink-" + blinkClassName + " ease-in-out infinite;\n" +
```

Comment:
since we need to keep the version of code not using animation for ie9, and almost all other browser support animation without prefixes https://caniuse.com/#feat=css-animation, it would be better to not use prefixes at all and put this into editor.css